### PR TITLE
docs(ee/dev-portal) Update dev portal auth docs for Routes/Services

### PR DESCRIPTION
**Pull request by Jlawlzz**
### Summary

Updates Kong Dev Portal authentication documentation to use Routes/Services
rather than then the now deprecated API entity

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

File: app/docs/enterprise/0.31-x/developer-portal/authentication.md [hunk 1]
```diff
@@ -61,23 +70,31 @@ curl -X POST http://127.0.0.1:8001/apis/portal-files/plugins \
 
 Now that we have setup authentication for your Dev Portal File API, your developers won't be able to access any of your files without credentials. What about access to **unauthenticated files,** files that have the flag `auth` set to `false`, such as landing pages, and the login form?
 
-Let's create another API to grant access to unauthenticated files:
+Let's create another Service/Route to grant access to unauthenticated files:
```
Comment (coopr) [inline app/docs/enterprise/0.31-x/developer-portal/authentication.md]: "Service/Route" -> "Service and Route" (for clarity and consistency)

File: app/docs/enterprise/0.31-x/developer-portal/authentication.md [hunk 2]
```diff
@@ -25,34 +25,43 @@ title: Authenticating the Developer Portal
...
+Route:
+
+```bash
+curl -i -X POST \
+  --url http://127.0.0.1:8001/services/portal-files/routes \
+  --data 'paths[]=/portal'
 ```
 
-Now that we created our API, update the following line in your Kong Configuration to let Kong know that the Public Dev Portal Files API should point to `:8000/portal` and restart Kong:
+Now that we created our Service/Route, update the following line in your Kong Configuration to let Kong know that the Public Dev Portal Files API should point to `:8000/portal` and restart Kong:
```
Comment (coopr) [inline app/docs/enterprise/0.31-x/developer-portal/authentication.md]: "Service/Route" -> "Service and Route"

File: app/docs/enterprise/0.31-x/developer-portal/authentication.md [hunk 3]
```diff
@@ -25,34 +25,43 @@ title: Authenticating the Developer Portal
 ## Enable Authentication
 
 
-First, create an API to proxy requests to the Public Dev Portal API:
+First, create a Service and corresponding Route to proxy requests to the Public Dev Portal API:
```
Comment (coopr) [inline app/docs/enterprise/0.31-x/developer-portal/authentication.md]: elsehwere you refer to the "Public Dev Portal Files API" (what a mouthful!) - this mention should use the same terminology for clarity.

**Comment by Jlawlzz** [comment]
[(1) Change documentation for Developer Portal Authentication to work with Routes and Services](https://trello.com/c/uQCaZgr1/656-1-change-documentation-for-developer-portal-authentication-to-work-with-routes-and-services)

**Pull request merged**